python/paddle/fluid/layers/nn.py

```diff
@@ -10560,3 +10561,52 @@ def tree_conv(nodes_vector,
     else:
         pre_activation = out
     return helper.append_activation(pre_activation)
+
+
+def npair_loss(anchor, positive, labels, l2_reg=0.002):
+    '''
+  **Npair Loss Layer**
+  
+  see http://www.nec-labs.com/uploads/images/Department-Images/MediaAnalytics/papers/nips16_npairmetriclearning.pdf
+
+  Npair loss requires paired data. Npair loss has two parts, the first part is L2 
+  regularizer on the embedding vector, the second part is cross entropy loss which
```

Comment:
```suggestion
  regularizer on the embedding vector; the second part is cross entropy loss which
```

Please use semicolon ';' to introduce equal level of parts.